py3status/modules/sysdata.py

```diff
@@ -257,21 +257,10 @@ def _calc_mem_info(self, unit, meminfo, memory):
         (used_mem, _) = self.py3.format_units(used_mem_kib * 1024, unit=unit)
         return total_mem, used_mem, used_mem_p, unit
 
-    def _get_mem(self, mem_unit, swap_unit, mem, swap):
-        meminfo = {}
-        result = {}
-
+    def _get_meminfo(self, head=49):
```

Comment:
this method is only called once and head is passed as `24`, why make it an argument instead of just hardcoding `24` on the line `fields = ...` ?

Reply:
I can do that. I don't like placing hardcoded args in the code. Reason for this is that if users added a new `mem` and/or `swap` function or two or three in the future and need more parameters beyond line 24, they can change the head there.

Reply:
Let's go for simplicity and readability, make default value of the argument `=24` then and don't pass it in the calling method

Reply:
Done.